roles/robottelo/tasks/ssh.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: 'Create ssh keys'
+  command: "ssh-keygen -b 2048 -t rsa -f {{ robottelo_ssh_key }} -q -N ''"
+  args:
+    creates: "{{ robottelo_ssh_key }}"
+
+- name: 'Add ssh key to authorized_keys'
+  copy:
+    remote_src: true
+    src: "{{ robottelo_ssh_key }}"
+    dest: /root/.ssh/authorized_keys
```

Comment:
Doesn't this `cp {{ robottelo_ssh_key }} /root/.ssh/authorized_keys` ? Don't we actually want to do `cat {{ robottelo_ssh_key }}.pub >> /root/.ssh/authorized_keys` ?

Reply:
What I'm saying is, authorized_keys isn't for private keys, and you can't just overwrite it, you have to append or lose all your other authorized keys.

Reply:
Oh you're right, the `src` should be `{{ robottelo_ssh_key }}.pub`. Good catch.

Reply:
We also shouldn't replace the file but simply ensure our public key is in there: https://github.com/theforeman/forklift/pull/754